ui/app/src/App.scss

```
@@ -25,7 +25,7 @@ code {
 }
 
 .tre-top-nav {
-  box-shadow: 0 1px 2px 0px #033d68;
+  box-shadow: 0 1px 2px 0px #6e7b2f;
```

```suggestion
  box-shadow: 0 1px 2px 0px #ea7600;
```

Suggestion is [UCL orange](https://www.ucl.ac.uk/cam/brand/guidelines/colour)